lib/matplotlib/tests/test_style.py

```diff
@@ -138,10 +137,9 @@ def test_context_with_union_of_dict_and_namedstyle():
 def test_context_with_badparam():
     original_value = 'gray'
     other_value = 'blue'
-    d = OrderedDict([(PARAM, original_value), ('badparam', None)])
     with style.context({PARAM: other_value}):
         assert mpl.rcParams[PARAM] == other_value
-        x = style.context([d])
+        x = style.context({PARAM: original_value, 'badparam': None})
```

Comment:
Do you need `x`, or can it just in the `with`?

Reply:
I suppose this was written to check that the error occurs when entering the context, and not when creating it.